File: INCLService/DatenM.pas
unit DatenM;

interface

uses
  Windows, Messages, SysUtils, Classes, Graphics, Controls,
  DB, MemDS, DBAccess, CO_DataBase, ADODB
  {$IFDEF ODAC}
    ,Ora;
  {$ELSE}
    {$IFNDEF NONUNI}
      ,Uni;
    {$ELSE}
      ;
    {$ENDIF}
  {$ENDIF}  

type
  TDaten = class(TDataModule)
    qSuch: TCO_Query;
    qUpdate: TCO_Query;
    qWerte: TCO_Query;
    qCount: TCO_Query;
    qCreateDB: TCO_Query;
    qSuch2: TCO_Query;
    qSuch4: TCO_Query;
    qIstwert: TCO_Query;
    qDurchlauf: TCO_Query;
    Database: TCO_Database;
    qTMP: TCO_Query;
    qSuch5: TCO_Query;
    qSuch3: TCO_Query;
    qUpdateS: TCO_Query;
    qLog: TCO_Query;
    qSetupPar: TCO_Query;
  private
  public
    Conn: Boolean;
  end;

var
  Daten: TDaten;

implementation

//uses Main;

{$R *.DFM}

end.

File: INCLService_CSharp/DatenM.cs
using System;

namespace INCLService_CSharp
{
    public class DatenM
    {
        public CO_Query qSuch { get; set; }
        public CO_Query qUpdate { get; set; }
        public CO_Query qWerte { get; set; }
        public CO_Query qCount { get; set; }
        public CO_Query qCreateDB { get; set; }
        public CO_Query qSuch2 { get; set; }
        public CO_Query qSuch4 { get; set; }
        public CO_Query qIstwert { get; set; }
        public CO_Query qDurchlauf { get; set; }
        public CO_Database Database { get; set; }
        public CO_Query qTMP { get; set; }
        public CO_Query qSuch5 { get; set; }
        public CO_Query qSuch3 { get; set; }
        public CO_Query qUpdateS { get; set; }
        public CO_Query qLog { get; set; }
        public CO_Query qSetupPar { get; set; }

        public bool Conn { get; set; }

        public static DatenM Instance { get; set; }
    }

    // Forward declarations for types used in this file
    public class CO_Database { }
    public class CO_Query { }
}
